util/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/AttributesUtil.java

```diff
@@ -118,9 +120,19 @@ public static String attributesToQidoPath(Attributes attrs, String... includeFie
       switch (attrs.getString(Tag.QueryRetrieveLevel)) {
         case "STUDY":
           qidoPath.append("studies?limit=" + STUDIES_SERIES_LIMIT + "&");
+          
+          if (FUZZY_MATCHING == "true") {
+            qidoPath.append("fuzzymatching=true" + "&");
+          }
+          
```

Comment:
We can also append fuzzy matching for an instances level retrieval, so I think this if statement could be moved outside of the switch statement, since it would apply for all query retrieve levels

Reply:
Actually I have tried adding it out of the switch, but I got a build failure for doing fuzzy matching at INSTANCE level. That's why I put it inside STUDY and SERIES level.

Reply:
What was the build failure? It might be a unit test that needs to be updated?

Reply:
Yes, it is a unit test.

expected: instances?limit=50000&00080061=MG&
but was : instances?limit=50000&fuzzymatching=true&00080061=MG&
	at com.google.cloud.healthcare.imaging.dicomadapter.AttributesUtilTest.testAttributesToQidoPathArray_manyModalitiesSet(AttributesUtilTest.java:105)

Reply:
Ok, can you please update the tests as well so that the build passes with fuzzy matching enabled for all three retrieval levels?

Reply:
Yes, I can do that!

Reply:
As per our discussion I have added the fuzzymatching attribute to Flags and referenced it in code. Let me know if everything looks OK after you reviewed my changes.

Reply:
could you please add unit tests to check what url is have orhave not fuzzymatching parameter on different retrieval levels?
also if you don't mind - squash commits into single one

Reply:
+1, please add a unit test

Reply:
Looks good overall though. I'm not sure why the contributor license agreement presubmit check isn't running, I'll need to figure that out before we submit this.

Reply:
added until test for fuzzy matching.